src/index.js

```diff
@@ -22,7 +22,7 @@ class UglifyJsPlugin {
       uglifyOptions = {},
       test = /\.js$/i,
       warningsFilter = () => true,
-      extractComments = false,
+      extractComments = true,
```

Comment:
This would produce a `name.js.LICENSE` file by default or just preserves the comments within the file ? Is it really necessary to produce this file by default instead of opt-in if needed ? 

`semver`-ness is also debatable here...

Reply:
@michael-ciniawsky yep, it is output `name.js.LICENSE` file, same as in `@0.4.6` version

Reply:
Was outputting the `file.js.LICENSE` the default in `v0.4.6` ?  I can't remember... 😄 and if so do we really want it to be the default ?

Reply:
@michael-ciniawsky ok let's test this :smile: i will put screen shot :+1:

Reply:
@michael-ciniawsky i was right :smile: 
![screenshot from 2018-03-07 22-08-27](https://user-images.githubusercontent.com/4567934/37112645-17339650-2254-11e8-9a34-a432dc83f758.png)

Reply:
```js
new UglifyJSPlugin({
    extractComments: true
})
```

Reply:
kk, maybe asking folks (e.g slack) about their preference (on by default Y/n ?) would be a good idea, if there is no response or people want then let's move ahead

Reply:
@michael-ciniawsky Good idea

Reply:
@michael-ciniawsky from slack
```
sokra [4:27 PM]
I would keep comments in code by default. LICENSE file by default could be confusing for users and the benefit is small...
```

Reply:
Let's do it?

Reply:
Ping @evilebottnawi @michael-ciniawsky

Reply:
@julmot WIP :+1: